heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -147,6 +147,31 @@
    */
   public static final String TOPOLOGY_COMPONENT_RAMMAP = "topology.component.rammap";
   /**
+   * Is this topology stateful? The format of this flag is boolean
+   */
+  public static final String TOPOLOGY_STATEFUL = "topology.stateful";
```

Comment:
Not really.
One could run a stateful topology in non stateful mode. i.e. I could write a IStatefulSpout/Bolt but may prefer to turn on and off the stateful config for maybe different use-cases or experimentation.